test/e2e/tests/test_certificate_authority.py

```diff
@@ -0,0 +1,82 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Integration tests for the PCA Certificate Authority.
+"""
+
+import pytest
+import time
+import logging
+from typing import Dict, Tuple
+
+from acktest.resources import random_suffix_name
+from acktest.k8s import resource as k8s
+from e2e import service_marker, CRD_GROUP, CRD_VERSION, load_acmpca_resource
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.tests.helper import ACMPCAValidator
+
+RESOURCE_PLURAL = "certificateAuthorities"
+
+DEFAULT_WAIT_AFTER_SECONDS = 5
+CREATE_WAIT_AFTER_SECONDS = 10
+DELETE_WAIT_AFTER_SECONDS = 10
+
+@service_marker
+@pytest.mark.canary
+class TestCertificateAuthority:
+
+    def test_create_delete(self, acmpca_client):
+        ca_name = "my-certificate-authority" 
```

Comment:
add random suffix to the name. you can look at examples in e2e tests of other ACK controllers.